### ISSUE TITLE
Mouse strain terms to add

### ISSUE THREAD

**Issue by ppavlidis** [EBISPOT/efo#2088]
Edit: Updated with revised table of requested terms

As discussed recently on efo-users.

> Hi Paul,
> 
> We include strains based on user request, which may be why you are seeing some strains but not others.
> 
> If you would like to request any rat strains to be added to EFO you may do so through our ticket request system: https://github.com/EBISPOT/efo/issues
> 
> Thank you,
> Zoë
> 

I'm attaching a CSV file that has a list of mouse strains (and one rat) that we'd like to see added to EFO. We have done some work to gather cross-reference MGI IDs and synonyms. We also tried to make sure these weren't already in EFO but it can sometimes be hard to tell because the links to Jax in EFO are all dead.

In addition to this list, it would be great if you could add  "Crl:CD1(ICR)" as a synonym of  the existing "[CD1 mus strain](http://www.ebi.ac.uk/efo/EFO_0005180)" term please!

Thanks very much

[strain-term-requests-v2.txt](https://github.com/EBISPOT/efo/files/12621768/strain-term-requests-v2.txt)

**Comment by ppavlidis** [comment]
I just realized that my proposed 129S/SvEv is probably the same as the existing [129S6](http://www.ebi.ac.uk/efo/EFO_0000598) as 129S/SvEv is listed as a synonym. Oops! (Strain nomenclature is no fun and the links in EFO to JAX are broken).

In addition have two more requests that need more explanation. I'm not sure these are represented anywhere already in EFO but we don't think so.

It would be useful to have terms for the common use of 129 x C57BL/6 hybrid mice that are inbred for some number of generations (e.g. 10, but it varies). There is a similar situation for 126S.

This is not the same as [129xC57BL/6](http://www.ebi.ac.uk/efo/EFO_0000601), which is apparently made specifically for one publication and seems to be F2 mice specifically. 

And for 126S x B6 there is [B6129SF1/J](https://www.jax.org/strain/101043) in JAX  but this refers to F1 only.

So it would be great to have something like B6129SFx and B6129Fx where Fx indicates an unspecified number of generations.

**Comment by ppavlidis** [comment]
Hi Zoe, I apologize but on further inspection we realize that at least one of the strains we asked for is already in EFO. Synonyms.

That's my request for AKR, which is the same as AKR/J (http://www.ebi.ac.uk/efo/EFO_0007726)

There could well be others like this, if you'd prefer we make a new list first after some more cleanup let me know, but if you are doing the same diligence maybe it's okay

**Comment by zoependlington** [comment]
Hi Paul! Please feel free to update with any already existing terms. I will double check all of them and ensure they're in EFO for our October release.

**Comment by ppavlidis** [comment]
> Hi Paul! Please feel free to update with any already existing terms. I will double check all of them and ensure they're in EFO for our October release.

I've attached a revised file which is in better shape.

**Comment by ppavlidis** [comment]
Hi Zoe, we also want to point out that the strain [WSB/Eij](http://www.ebi.ac.uk/efo/EFO_0007731) should have the exact synonym WSB. My curators have seen this in the literature and even the definition in EFO/Jax uses that synonym. https://www.jax.org/strain/001145

**Comment by ppavlidis** [comment]
Hi, I don't see the new terms in the 3.59.0 release, is that expected?

**Comment by zoependlington** [comment]
Hi @ppavlidis, apologies I have been on leave. The terms will be available after our November release. I apologise again for the delay.

**Comment by zoependlington** [comment]
Hi @ppavlidis, these terms have now been added in EFO with IDs listed below. These will be available after the 3.60.0 release on 15th November.

ID | LABEL
-- | --
EFO_0022385 | Long Evans
EFO_0022387 | SJL/J
EFO_0022395 | C57BL/10J
EFO_0022396 | DBA/2
EFO_0022397 | C57BL/6
EFO_0022412 | SJL
EFO_0022419 | DBA
EFO_0022463 | CBA
EFO_0022464 | C3H/He
EFO_0022465 | C3HeB/FeJ
EFO_0022466 | A/HeJ
EFO_0022467 | FVB/N
EFO_0022468 | C3H
EFO_0022469 | DBA/1LacJ
EFO_0022470 | NOD/Shi
EFO_0022471 | NOR
EFO_0022472 | CFW
EFO_0022474 | 129S2/SvPas
EFO_0022475 | NOR/LtJ
EFO_0022476 | NOR1/Lt
EFO_0022477 | NOD/ShiLt
EFO_0022478 | BXD
EFO_0022473 | 129S/Sv
EFO_0022479 | 129T2/SvEmsJ
EFO_0022480 | 129P3/J
EFO_0022481 | 129S1/SvlmJ
EFO_0022482 | 129S/SvEv
EFO_0022483 | FVB
EFO_0022484 | MMTV-PyMT

As for your query about Fx specifying generations, I would be happy to add this into EFO with an explanation in the definition regarding what the "Fx" stands for - could you request those specific terms in a new ticket please? Thank you!

**Comment by ppavlidis** [comment]
Super, thanks Zoe. 

We may have another batch of these in the near future, I can clarify about B6129SFx etc. at that time.

**Comment by ppavlidis** [comment]
Hi Zoe, sorry but in reviewing this my curators realized there were already entities in EFO for two of the strains we asked for,  DBA/2 (it's a synonym for http://www.ebi.ac.uk/efo/EFO_0005184) and 129S1/SvlmJ  (http://www.ebi.ac.uk/efo/EFO_0001332)

So you can remove these but it would be good to add DBA/2 as an exact synonym for DBA2?

Again sorry for this oversight, we'll be more careful if/when we have more terms to suggest.

**Comment by zoependlington** [comment]
No worries, I'll remove the new term and add the synonym! Thanks for catching it!

**Comment by zoependlington** [comment]
Moving this to "Done" bucket as the duplicates have been removed and synonym added - this will be available after our November (15th) release.
